packages/storefront-api-client/README.md

```diff
@@ -338,6 +338,53 @@ if (response.ok) {
 }
 ```
 
+## Typing variables and return objects
+
+This client is compatible with the `@shopify/api-codegen-preset` package.
+You can use that package to create types from your operations with the [Codegen CLI](https://www.graphql-cli.com/codegen/).
+
+There are different ways to [configure codegen](https://github.com/Shopify/shopify-api-js/tree/main/packages/api-codegen-preset#configuration) with it, but the simplest way is to:
+
+1. Add the preset package as a dev dependency to your project, for example:
+    ```bash
+    npm install --save-dev @shopify/api-codegen-preset
+    ```
+1. Create a `.graphqlrc.ts` file in your root containing:
+    ```ts
+    import { ApiType, shopifyApiProject } from "@shopify/api-codegen-preset";
+
+    export default {
+      schema: "https://shopify.dev/storefront-graphql-direct-proxy",
+      documents: ["*.ts", "!node_modules"],
+      projects: {
+        default: shopifyApiProject({
+          apiType: ApiType.Storefront,
```

Comment:
Similar to the Admin API comment but we should explicitly set the `apiVersion` here since it's a required field for the api clients anyways and not setting it will likely actually cause more problems in the long run.